README.md

```diff
@@ -88,13 +88,14 @@ See the section about [running tests](https://facebook.github.io/create-react-ap
 
 ## Environment Variables
 
-| Key                                | Required      | Default value      | Description |
-| ------------- | ------------- | ------------- | ------------- |
-| REACT_APP_API_DOMAIN               | yes | null | MindLogger Backend API base URL |
-| REACT_APP_ENV                      | no | null | Environment to run the app in (`prod` or `staging`)  |
-| REACT_APP_DEVELOP_BUILD_VERSION    | no | null | Footer app build number |
-| REACT_APP_MIXPANEL_TOKEN           | no | null | Mixpanel token |
-| REACT_APP_LAUNCHDARKLY_CLIENT_ID           | no | null | LaunchDarkly client key to choose target environment |
+| Key                              | Required | Default value | Description                                          |
+|----------------------------------|----------|---------------|------------------------------------------------------|
+| REACT_APP_API_DOMAIN             | yes      | null          | MindLogger Backend API base URL                      |
+| REACT_APP_WEB_URI                | yes      | null          | Base URL of the MindLogger respondent web app        |
```

Comment:
I left this one null because the default value for `REACT_APP_API_DOMAIN` is also null, even though there's a value for it in `.env.example`. I assumed this was because the way we run the app locally varies, but putting a value here makes sense